commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/wishlist/CommerceWishListResource.java

```diff
@@ -71,11 +75,36 @@ public Response addWishListItem(
 				_commerceWishListService.getDefaultCommerceWishList(
 					groupId, _portal.getUserId(httpServletRequest));
 
-			_commerceWishListItemService.addCommerceWishListItem(
-				commerceWishList.getCommerceWishListId(), cpDefinitionId,
-				cpInstanceId, options, serviceContext);
+			CPDefinition cpDefinition = _cpDefinitionService.getCPDefinition(
+				cpDefinitionId);
 
-			wishListItemUpdated.setSuccess(true);
+			CPInstance cpInstance = _cpInstanceService.getCPInstance(
```

Comment:
@alecsloan This can be null, you can add to a wishlist a porduct without a sku, in case of multiple option

Reply:
see #752